classes/01class/exercises/c01-git02/feernandobraga/README.md

```diff
@@ -0,0 +1,26 @@
+# c01-git02
+
+## Answers
+
+1. Is it possible to see the inner commit of squashed commit with `git rebase`?
+
+```
+If you are trying to see the local commits, then yes. You could use git reflog to retrieve information about commits (and basically all the movements of the HEAD) for your device.
```

Comment:
right, and even in the remote repo it is possible to checkout an inner commit from a squash if you know the commit sha.